accounting/eval_preload.go

```diff
@@ -34,9 +36,15 @@ func addToCreatorsRequest(stxnad *transactions.SignedTxnWithAD, assetsReq map[ba
 			appsReq[fields.ApplicationID] = struct{}{}
 		}
 		for _, index := range fields.ForeignApps {
+			if index > basics.AppIndex(math.MaxInt64) {
```

Comment:
why do we use math.MaxInt64 instead of math.MaxUint64?

Reply:
This is related to the `BIGINT` -> `NUMERIC(20)` issue, algod/sqlite has a similar problem that this works around.